app/views/recipes/index.html.erb

```diff
@@ -1,14 +1,25 @@
-<p style="color: green"><%= notice %></p>
+<div class="w-full">
+  <% if notice.present? %>
+    <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+  <% end %>
 
-<h1>Recipes</h1>
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-4xl">Recipes</h1>
+    <%= link_to 'New recipe', new_recipe_path, class: "rounded-lg py-3 px-5 bg-blue-600 text-white block font-medium" %>
+  </div>
 
-<div id="recipes">
-  <% @recipes.each do |recipe| %>
-    <%= render recipe %>
-    <p>
-      <%= link_to "Show this recipe", recipe %>
-    </p>
-  <% end %>
-</div>
+  <div id="recipes" class="min-w-full flex flex-col gap-4 items-center justify-center mt-10">
+    <% @recipes.each do |recipe| %>
+      <div class="flex justify-around items-start gap-10 h-[170px] w-[700px] rounded-md p-8 bg-slate-200">
+        <div class="flex flex-col gap-10 justify-between">
+          <h2 class="font-bold text-2xl"><%= recipe.name %></h2>
+          <%= link_to "Remove", recipe_path(recipe), method: :delete, data: { confirm: 'Are you sure?' }, class: "text-red-600 ml-2 underline"  %>
```

Comment:
- [ ] Great job Adding the `tailwind` CSS style, However, when I tried to delete each recipe it doesn't delete instead goes to the recipe details path. but expected behavior should delete the recipe on the same page means the recipe#index page 

- [ ] I explained it in the video:

https://github.com/peterdtitan/food-recipe-app/assets/118761303/eed77b46-b1b3-4db7-9859-e90fe8ed14f6